backend/storage/StorageContext.ts

```diff
@@ -129,6 +130,34 @@ export class StorageContext {
     return row != null ? dbTransformations.createMatchFromDbRow(row) : null
   }
 
+  async getAllGames(): Promise<Array<Game>> {
+    let rows
+    try {
+      rows = await this.transaction.executeQuery(dbQueries.selectAllGames, [])
+    } catch (error) {
+      console.error(error)
+      throw new Error('Unable to retrieve games')
+    }
+    return rows.map(dbTransformations.createGameFromDbRow)
+  }
+
+  async insertGame(game: GameData): Promise<Game> {
+    let row
+    try {
+      row = await this.transaction.executeSingleResultQuery(dbQueries.insertGame, [
+        game.name,
+        game.description,
+      ])
+    } catch (error) {
+      console.error(error)
+      if (dbErrors.isUniqueViolation(error)) {
+        throw new ConflictError('Game already exists')
+      }
+      throw new Error('Unable to insert new game')
+    }
+    return row !== null ? dbTransformations.createGameFromDbRow(row): null
```

Comment:
`x !== null ? FOO : null` could we simplified into `x && FOO`

Reply:
fixed in https://github.com/salsita/foosball-rating/pull/113/commits/d8bb46a15e9ce4e8733aba65b651d0ec0eece570